<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>94</height>
       </size>
      </property>
      <property name="title">
       <string>Image Tools</string>
      </property>
      <widget class="QPushButton" name="subtractButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Subtract</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ratioButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ratio</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="outlineCheck">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>LUT Slider</string>
       </property>
      </widget>
      <widget class="QPushButton" name="voltageButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Voltage Test</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>248</width>
        <height>106</height>
       </size>
      </property>
      <property name="title">
       <string>ROI Tools</string>
      </property>
      <widget class="QPushButton" name="plotAllButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot All</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lineButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/line.png</normaloff>../images/line.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="rectangleButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/rect.png</normaloff>../images/rect.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="freehandButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/freehand.png</normaloff>../images/freehand.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pointButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../images/point.png</normaloff>../images/point.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="importROIButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Import ROIs</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exportROIButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Export ROIs</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>Trace Analysis</string>
      </property>
      <widget class="QPushButton" name="lineMeasureButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Line Measure</string>
       </property>
      </widget>
      <widget class="QPushButton" name="showTraceButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Average Movie Trace</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="newWindowCheck">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot Traces in New Window</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Close Windows</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Movie"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionChange_Internal_Data_type"/>
   </widget>
   <widget class="QMenu" name="menuMath">
    <property name="title">
     <string>Math</string>
    </property>
    <addaction name="actionSubtract"/>
    <addaction name="actionMultiply"/>
    <addaction name="actionRatio"/>
    <addaction name="actionAbsolute_Value"/>
    <addaction name="actionSubtract_Trace"/>
   </widget>
   <widget class="QMenu" name="menuROIs">
    <property name="title">
     <string>ROIs</string>
    </property>
    <addaction name="actionImport_ROI_File"/>
    <addaction name="actionExport_ROIs"/>
    <addaction name="actionClear_ROIs"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Points"/>
    <addaction name="actionLoad_Points"/>
   </widget>
   <widget class="QMenu" name="menuScripts">
    <property name="title">
     <string>Scripts</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuROIs"/>
   <addaction name="menuMath"/>
   <addaction name="menuScripts"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSubtract">
   <property name="text">
    <string>Subtract</string>
   </property>
  </action>
  <action name="actionMultiply">
   <property name="text">
    <string>Multiply</string>
   </property>
  </action>
  <action name="actionPower">
   <property name="text">
    <string>Power</string>
   </property>
  </action>
  <action name="actionRatio">
   <property name="text">
    <string>Ratio By Baseline</string>
   </property>
  </action>
  <action name="actionAbsolute_Value">
   <property name="text">
    <string>Absolute Value</string>
   </property>
  </action>
  <action name="actionSubtract_Trace">
   <property name="text">
    <string>Subtract Trace</string>
   </property>
  </action>
  <action name="actionSave_Movie">
   <property name="text">
    <string>Save Movie</string>
   </property>
  </action>
  <action name="actionImport_ROI_File">
   <property name="text">
    <string>Import ROI File</string>
   </property>
  </action>
  <action name="actionExport_ROIs">
   <property name="text">
    <string>Export ROIs</string>
   </property>
  </action>
  <action name="actionClear_ROIs">
   <property name="text">
    <string>Clear ROIs</string>
   </property>
  </action>
  <action name="actionLoad_Points">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionSave_Points">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionChange_Internal_Data_type">
   <property name="text">
    <string>Change Internal Data type</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionLine_Distance">
   <property name="text">
    <string>Line Distance</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
